docs/kf_uids/uid_proposal.md

```diff
@@ -0,0 +1,38 @@
+# Proposal
+
+Based on [uid_considerations.md](uid_considerations.md) and [uid_migration.md](uid_migration.md)
+
+## 64 bit keys
+
+### Nodeless (much better than what we have now)
+
+* 31 bit seconds since custom epoch (now until 68 years from now)
+* 33 bit randomized counter (8.5 billion keys for every second)
+* Cross fingers
+
```

Comment:
One thing that came up is which bits go where, this is mostly an aesthetic choice. Three options:

1. Seconds bits and then randomized bits
2. Randomized and then second bits
3. Interleaving bits

3 would look mostly random. 1 would make the identifiers easily sortable/scannable with a sense of time. 2 would look random, but a careful eye would see the last bit is incremental. Each has pros/cons in terms of user usage, examples might help @fiendish ?

If we don't do a mapping table for migration, there might be other considerations about using zero time to migrate the previous identifiers.